node.gyp

```diff
@@ -285,8 +289,6 @@
         'src/node.h',
         'src/node_buffer.h',
         'src/node_constants.h',
-        'src/node_file.h',
-        'src/node_http_parser.h',
```

Comment:
Are you sure these two headers files should be removed? Or is it another commit that can be removed?